### PR TITLE
ENG-14619-voltsql-improvement-2

### PR DESCRIPTION
**Pull request by Chaozz**
This PR is a mixed fix for ENG-14619 and ENG-14620.
Also improve the layout of the bottom toolbar.

### CODE REVIEW

File: lib/python/voltsql/voltsql/voltliterals/volt_literals.json [hunk 1]
```diff
@@ -89,14 +90,15 @@
         ],
         "PRIMARY": [],
         "RECURSIVE": [],
+        "REMOVE CLASSES": [],
```
Comment (yzhang1991) [inline lib/python/voltsql/voltsql/voltliterals/volt_literals.json]: Is there a LOAD CLASSES?

File: lib/python/voltsql/voltsql/voltliterals/volt_literals.json [hunk 2]
```diff
@@ -113,7 +115,10 @@
         "VIEW": [],
         "WHEN": [],
         "WHERE": [],
-        "WITH": []
+        "WITH": [],
+        "QUIT": [],
+        "REFRESH": [],
```
Comment (yzhang1991) [inline lib/python/voltsql/voltsql/voltliterals/volt_literals.json]: Where do we use refresh?

File: lib/python/voltsql/voltsql/voltliterals/volt_literals.json [hunk 3]
```diff
@@ -251,17 +256,16 @@
     ],
     "datatypes": [
         "BIGINT",
-        "BOOLEAN",
-        "CHAR",
-        "DATE",
-        "DOUBLE PRECISION",
-        "INT",
```
Comment (yzhang1991) [inline lib/python/voltsql/voltsql/voltliterals/volt_literals.json]: INT is valid

File: lib/python/voltsql/voltsql/voltliterals/volt_literals.json (hunk 1 above)
Comment (Chaozz) [reply lib/python/voltsql/voltsql/voltliterals/volt_literals.json]: Yes, there is.

File: lib/python/voltsql/voltsql/voltliterals/volt_literals.json (hunk 2 above)
Comment (Chaozz) [reply lib/python/voltsql/voltsql/voltliterals/volt_literals.json]: I change previous voltsql special command UPDATE to REFRESH(cause UPDATE is already a keyword). It is can be used to force the voltsql refreshing the schemas from catalog.

**Pull request merged**